examples/all_the_things.py

```diff
@@ -109,5 +111,5 @@ def attachments(test):
   #  with open('my_private_key.json', 'r') as json_file:
   #    test.AddOutputCallbacks(mfg_inspector.UploadToMfgInspector.from_json(
   #        json.load(json_file)))
-
+  test.Configure(teardown_function=teardown)
```

Comment:
if you're going to add a call to `.Configure()` here, might as well also disable `http_api` with `http_port=None`.